jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/util/ReflectionHelper.java

```diff
@@ -42,28 +45,28 @@ public ReflectionHelper(RuleFactory ruleFactory) {
   }
 
   public JType getSuperType(String nodeName, JsonNode node, JPackage jPackage, Schema schema) {
-    if (node.has("extends") && node.has("extendsJavaClass")) {
-      throw new IllegalStateException("'extends' and 'extendsJavaClass' defined simultaneously");
+    if (hasExtensionProperty(node, "extends") && hasExtensionProperty(node, "extendsJavaClass")) {
+      throw new IllegalStateException("'(x-)extends' and '(x-)extendsJavaClass' defined simultaneously");
```

Comment:
not sure its worth it, but instead of (x-), can substitute for getExtensionPropertyNameUsed(..)

Reply:
Changed (and squashed away per project maintainer's guideilines)